charts/images.yaml

```diff
@@ -27,6 +27,10 @@ images:
   sourceRepository: github.com/gardener/machine-controller-manager
   repository: eu.gcr.io/gardener-project/gardener/machine-controller-manager
   tag: "v0.36.0"
+- name: machine-controller-manager-provider-aws
+  sourceRepository: github.com/gardener/machine-controller-manager-provider-aws
+  repository: eu.gcr.io/gardener-project/gardener/machine-controller-manager-provider-aws
```

Comment:
Can we update the README.md of `github.com/gardener/machine-controller-manager-provider-aws`. It still states `Work in progress.` and it sounds odd to switch to something that is still WIP.

Reply:
Sorry about that. Fixed it here - https://github.com/gardener/machine-controller-manager-provider-aws.

Reply:
Looks very good now. Thank you.